.gitignore

```diff
@@ -60,3 +60,6 @@ bin/
 gen/
 keystore/
 google-services.json
+
+#iOS related
+GoogleService-Info.plist
```

Comment:
.gitignore for iOS exists in iOS folder for the flutter project, can you have a look, also the path should matter right ?

Reply:
path is base iOS project folder.
Added.